rfc2544/README.md

```diff
@@ -49,27 +49,53 @@ podman pod create -p 50051:50051 -n trafficgen
 podman run -d --rm --privileged -v /dev:/dev -v /sys:/sys -v /lib/modules:/lib/modules --cpuset-cpus 4-11 --pod trafficgen -e pci_list=0000:03:00.0,0000:03:00.1  docker.io/cscojianzhan/trafficgen /root/trafficgen_entry.sh server
```

Comment:
Now that  CMD ["/root/trafficgen_entry.sh", "server"] is used, the README should reflect that, 1) how to kick off a manual run with "/root/trafficgen_entry.sh start" 2) how to kick off the rest api server without the extra argument

Reply:
Updated in latest commit